applications/GeoMechanicsApplication/tests/test_rotation_with_moving_load.py

```diff
@@ -39,7 +38,7 @@ def test_rotation_with_moving_load(self):
         # This test is a regression test, if rotation is not added to the
         # newmark upw scheme as a variable that needs to be predicted and
         # updated, the results will be different. TOTAL_ROTATION should be equal to ROTATION
-        self.assertAlmostEqual(-0.000177858, rotations_for_node_1[2])
+        self.assertAlmostEqual(-0.000177858, rotations_for_node_1_time_1[2])
 
         self.assertAlmostEqual(-0.000177858, total_rotations_for_node_1_time_1[2])
         self.assertAlmostEqual(-0.000238568, total_rotations_for_node_1_time_1_5[2])
```

Comment:
This is completely irrelevant to this PR but maybe the name below makes more sense:
```suggestion
        self.assertAlmostEqual(-0.000238568, total_rotations_for_node1_time1_5[2])
```
Then when we have this number 5 at the end of it, which makes a bit more sense? But feel free to ignore this because this is completely irrelevant to this PR I just saw you have changed the variable name, so that's why :D